### PR TITLE
Button - added primary styles for borderless buttons

### PR DESCRIPTION
**Pull request by MichaelArestad**
#### Before
image

#### After
image

image

### CODE REVIEW

File: client/components/button/style.scss [hunk 1]
```diff
@@ -211,6 +211,20 @@ button {
 		}
 	}
 
+	&.is-primary {
```
Comment (gwwar) [inline client/components/button/style.scss]: This section is getting pretty long so consider breaking this out to:
```
.button.is-borderless.is-primary {
```

Mainly this is an optional readability thing. For example I had to load more line of this file to see what this was modifying.

**Comment by gwwar** [comment]
Code changes look reasonable, but I'll defer to a design +1 for updates here.

File: client/components/button/style.scss (hunk 1 above)
Comment (MichaelArestad) [reply client/components/button/style.scss]: I agree. I had to fight the urge to refactor the whole thing for a small change. I think the example needs refactoring as well and the ability to cycle through button styles.

**Comment by drw158** [comment]
Looks good to me

**Pull request merged**